tools/testing/build_and_run_tests.sh

```diff
@@ -34,5 +34,5 @@ if ! [ -x "$(command -v nvidia-smi)" ]; then
   EXTRA_ARGS="-n auto"
 fi
 
-
+bazel clean
```

Comment:
@seanpmorgan I want to remove this but if I don't remove this it find in the cache some so that I don't undertands:

```
tensorflow.python.framework.errors_impl.NotFoundError: /addons/bazel-bin/tensorflow_addons/custom_ops/layers/lib_correlation_cost_ops_gpu.so: undefined symbol: _ZNK10tensorflow6Tensor21CheckTypeAndIsAlignedENS_8DataTypeE
```

In the local build I don't find any `lib_correlation_cost_ops_gpu.so`